Incident Wash-Up

```diff
@@ -0,0 +1,19 @@
+Incident Wash-Ups should be used to discuss issues that have affected our customers and how we can help to avoid similar situations from occurring in the future. The Wash-Up should include all relevant parties at their earliest opportunity after the event. 
+
+Please use this format for all Incident Wash-Ups so that we have a consistent set of data on how we handle situations. Please add all notes from Incident Wash-Ups to the Confluence space (https://holidayextras.jira.com/wiki/display/WEB/Incident+Wash-Ups), for reference.
```

Comment:
@notABluesSinger is on holiday for 2 weeks.

I don't think we use confluence - as we have a separate Jira system would this result in an extra cost?

Incident Recovery Jiras are setup as a separate project on our Jira, which has the questions as part of the Jira creation form.

![incident recovery](https://cloud.githubusercontent.com/assets/46255/9492284/131f8874-4bee-11e5-84ef-386a7aaa9f17.png)

If you have access to our Jira you can see all issues listed here
https://hxshortbreaks.atlassian.net/projects/IR/issues/IR-18?filter=allissues